### ISSUE TITLE
Create different classes of references

### ISSUE THREAD

**Issue by frivoal** [tabatkins/bikeshed#1401]
The W3C Process distinguishes between [publicly available references](https://w3c.github.io/w3process/#public-refs) and [references with member-only access](https://w3c.github.io/w3process/#member-refs).

I'm converting that document to Bikeshed, and it would be nice to be able preserve that distinction in the bikeshed-generated bibliography.

Something like a "ref-category" key in the JSON placed in `<pre class=biblio>` blocks or in biblio.json, together with some other bikeshed specific metadata for the title and opening sentense of the section should do it.

**Comment by tabatkins** [comment]
Hm, interesting. I'd be okay with doing something for this; at least being able to mark a biblio as "private/restricted" and have it segregated in the index.

**Comment by jyasskin** [comment]
The Process appears to have lost that distinction. Do we still need this?

**Comment by frivoal** [comment]
I don't think it is a must have, but the Process continues to have refencnes to some documents which aren't public. Being able to call them out / list them separately still sounds like a good—if low priority—idea to me.
